packages/styled-components/src/sheet/test/GroupedTag.test.js

```diff
@@ -2,78 +2,82 @@
 
 import { VirtualTag } from '../Tag';
 import { DefaultGroupedTag } from '../GroupedTag';
+import { WindowedTag } from '../WindowedTag';
 
-let tag;
-let groupedTag;
+const describeGroupedTag = makeTagAndGroupedTag => {
```

Comment:
I'm not super familiar with Flow typing, so I wasn't sure if it was needed to try and add types for this function `makeTagAndGroupedTag`. If y'all want them I can try to figure it out though :)

Reply:
I think it’s fine, those are just tests 👍